rest_framework/renderers.py

```diff
@@ -790,3 +791,17 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
                     "test case." % key
                 )
         return encode_multipart(self.BOUNDARY, data)
+
+
+class CoreJSONRenderer(BaseRenderer):
```

Comment:
Kinda. I see `coreapi` as a foundational thing here, so it's a bit different.

The various types of schema and docs that you can use it to generate will be third party, yup.
So eg we can have third party packages for schema formats: Swagger / API Blueprint / JSON Hyperschema, for docs templates driven by a coreapi.Document object, and for various hypermedia styles.

Using `coreapi` ensures that we're able to provide a common interface for all of those, so I don't have any great issues with pulling it in. Having said that, we *could* push for it to be third-party, if we wanted to.